src/Sylius/Component/Grid/Filter/EntityFilter.php

```diff
@@ -29,8 +29,16 @@ public function apply(DataSourceInterface $dataSource, $name, $data, array $opti
         }
 
         $field = isset($options['field']) ? $options['field'] : $name;
+        $fields = isset($options['fields']) ? $options['fields'] : [$field];
```

Comment:
Yup, let's say I have the following grid configuration included from thrid-party vendor:

```yaml
sylius_grid:
  my_grid:
    # ...
    filters:
      field:
        type: entity
        options:
          fields: [first_field, second_field]
```

Then I want to overwrite the filtering to use the first field only:

```
sylius_grid:
  my_grid:
    # ...
    filters:
      field:
        type: entity
        options:
          field: first_field
```

This merges into:

```yaml
sylius_grid:
  my_grid:
    # ...
    filters:
      field:
        type: entity
        options:
          field: first_field
          fields: [first_field, second_field]
```

And I am not expecting, that GridBundle will still look for that value in `second_field`.